src/status_im/events.cljs

```diff
@@ -2135,3 +2134,54 @@
  :shake-event
  (fn [cofx _]
    (logging/show-logs-dialog cofx)))
+
+;; ENS
+
+(handlers/register-handler-fx
+ :ens/clear-cache-and-navigate-back
+ (fn [{:keys [db] :as cofx} _]
```

Comment:
i think Eric meant that we can move all these events to core, with new fx/defn we don't need to register events manually anymore, could you please move all these events to core? should be a simple change, thanks

Reply:
:+1: